recipes/igraph/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{%set name = "igraph" %}
+{%set version = "0.7.1" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "d978030e27369bf698f3816ab70aa9141e9baf81c56cc4f55efbe5489b46b0df" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://igraph.org/nightly/get/c/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }} 
+  patches:
+    - b215195214abb8355121a8430c86df04e1f00eae.patch  # fixes igraph/igraph#906
+
+build:
+  skip: True  # [win or osx]
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - gmp
+    - libxml2
+    - pkg-config
+  run:
+    - gmp 
+    - libxml2
+
+test:
```

Comment:
Are there any file existence tests we can add (e.g. headers, libraries, binaries, etc.)?